codehs/scratchjr
# Add ability to see project name in scratch jr

Description:
needs to be done with https://github.com/codehs/codehs/pull/31601

Allow students/teachers to change the name of sandbox programs in the scratchjr editor

Issues/Tickets:
- fixes ENG-4494

Screenshots:
https://www.loom.com/share/ec5e15145e234922bec4a9ef3bba2a48?sid=47853f21-6090-4255-8556-996296e71543

Steps to Test:
create a scratchjr sandbox program and change the name in the editor, make sure the page refreshes and the url is updated with the right title

Local URLs:
- http://localhost:8000/student/playground/5247638?ref=3

Staging URLs:
- https://staging.codehs.com/student/playground/5247638?ref=3

Live URLs:
- https://codehs.com/student/playground/5247638?ref=3
